package.json

```diff
@@ -187,7 +187,9 @@
     "eslint-plugin-react-hooks": "^5.2.0",
     "eslint-plugin-react-refresh": "^0.4.20",
     "eslint-plugin-simple-import-sort": "^12.1.1",
+    "front-matter": "^4.0.2",
     "globals": "^16.3.0",
+    "html-to-react": "^1.7.0",
```

Comment:
what are these and why you added them?

Reply:
These deps were used in `scripts/` but weren't actually declared anywhere, it popped up once I changed these files to `.cjs` and knip started checking them.